src/miner/[minerId].md

```diff
@@ -0,0 +1,71 @@
+```js
+import { LineGraph } from "../components/line-graph.js";
+import { getDateXDaysAgo } from "../utils/date-utils.js";
+const data = FileAttachment(`../data/${observable.params.minerId}-spark-rsr-summary.json`).json();
+```
+
+<div class="hero">
+  <body><a href="/"><img src="../media/spark-logomark-blue-with-bbox.png" alt="Spark Logo" width="300" /></a><body>
+    <h2>Dashboard Beta</h2>
+    <body><a href="https://filspark.com/dashboard" target="_blank" rel="noopener noreferrer">(Click here for Legacy Spark Grafana Dashboard)</a><body>
+</div>
+
+<h4>Miner Spark RSR Summary</h4>
+<body>This section shows the miner Spark Retrieval Success Rate Score summary. You can adjust the date range. Records start on the 7th April 2024.</body>
+
+
+```js
+const start = view(Inputs.date({label: "Start", value: getDateXDaysAgo(180) }));
+const end = view(Inputs.date({label: "End", value: getDateXDaysAgo(1) }));
+```
+
+<h3>Data for Miner ID: ${observable.params.minerId}</h3>
```

Comment:
```suggestion
<h3>Stats for ${observable.params.minerId}</h3>
```

Data is very generic, plus we're not really supposed to call providers miners any more.

Reply:
Thanks for heads up, I might update the route too (change from /miner to /provider)?

Reply:
Yes please